### PR TITLE
Make all modal tooltips left-bottom positioned

### PR DESCRIPTION
**Pull request by vblagoje**
Do not integrate @ryanemerson . This is just a proposal for you to consider!

### CODE REVIEW

File: src/module/cache-container/config/security/view/add-role-modal.html [hunk 1]
```diff
@@ -13,8 +13,8 @@ <h4 class="modal-title" id-generator="add_role_modal.title" translate>PERMISSION
           <input id-generator="role.name" type="text" class="form-control" ng-model="ctrl.role.name" ng-readonly="!ctrl.isNewRole">
         </div>
         <div class="col-md-2 pull-left" style="padding-left: 0">
-          <i class="fa fa-info-circle ng-scope" tooltip="The security role name" tooltip-trigger="" tooltip-animation="true"
-             tooltip-placement="top"></i>
+          <i class="fa fa-info-circle ng-scope" tooltip="The security role name"
+             tooltip-placement="left-bottom"></i>
```
Comment (ryanemerson) [inline src/module/cache-container/config/security/view/add-role-modal.html]: (Applies to master as well) This tooltip doesn't display as it still uses `tooltip=` when it should be `uib-tooltip=`.

**Comment by ryanemerson** [comment]
@vblagoje I have nothing against the idea, however you still need to update the following files:

src/components/forms/field-info/view/field-info.html
src/components/cache-security/view/cache-security.html

Is there a reason for the change? Is having top aligned tooltips causing issues?

File: src/module/cache-container/config/security/view/add-role-modal.html (hunk 1 above)
Comment (ryanemerson) [inline src/module/cache-container/config/security/view/add-role-modal.html]: Based upon a quick grep, this also applies to `src/components/cache-security/view/cache-security.html`

**Comment by vblagoje** [comment]
@ryanemerson I wanted to have tooltips in modals (not field info and security) align left as we have made a few of them already align left in other modals (a commit or so ago). This way we stay consistent. Ideally we could make this a setting in a config. Will explore this a bit more after ISPN-7425 is done

**Comment by ryanemerson** [comment]
@vblagoje Yeah having it configurable would be cool. A thought for the future, we could configure UI preferences like this via URL parameters that are applicable to all pages. As we use a base url that all pages inherit from, we would only have to write the resolution code once and then we could just inject a "UIProperties" object into the relevant controllers and the templates would call something like `$ctrl.uiProps.getTooltipAlignment()`. If no url params are specified, then we simply use default values.

**Comment by vblagoje** [comment]
@ryanemerson I think I know what you mean but how would you store these UI preferences? One global default and then each page inherits these and overwrites them? How would $ctrl.uiProps.getTooltipAlignment() resolve the value? We need to write this UIProps class to do the value resolution!

**Comment by ryanemerson** [comment]
@vblagoje You would store them in an object, which is then injected into controllers where required. For every page load, you a new instance of this object is created in the resolve method of the root state and it can then be injected as required. 

We could create a Immutable UIProperties class which stores all of the values and exposes them via getter methods.This class could then provide a static create method, which takes a $stateParams object as its only arg and creates a new UIProperties instance using the current URL parameters to initialize the instance's values.

**Pull request closed without merging**